docs/users/emr-erp.md

```diff
@@ -62,6 +62,106 @@ flowchart LR
 - [OpenMRS patient → Odoo customer](#openmrs-patient-odoo-customer)
 - [OpenMRS billable item ⭆ Odoo quotation](#openmrs-billable-items-odoo-quotation)
 
-#### Configurability
+#### Options
 
-:construction: tbc
\ No newline at end of file
+:construction: tbc
+
+## Data Flows between ERPNext and OpenMRS 
+
+### OpenMRS Patient → ERPNext Customer
+
+#### Summary
+A patient in OpenMRS is synchronized as a customer in ERPNext.
+
+#### Main Flow
+``` mermaid
+flowchart LR
+    a["OpenMRS patient"]-- 1-to-1 -->b["ERPNext customer"]
+```
+
+#### Prerequisite Flows
+
+None
+
+#### Options
+
+=== "Behaviour"
+
+    - <small>**default**</small> &nbsp; An OpenMRS patient is synchronised as an ERPNext customer when a first billable item is ordered from OpenMRS.
+    - <small>_optional_</small> &nbsp; An OpenMRS patient is always synchronised as an ERPNext customer.
+
+=== "Configuration"
+
+    - File:<br/>`ozone/distro/configs/eip-erpnext-openmrs/application.properties`
+    - Property name:<br/>`erpnext.openmrs.enable.patient.sync`
+    - Possible values:<br/>`false` (<small>**default**</small>), `true`
```

Comment:
Just fyi, my take at trying to documents a flow's options. Let me know what you think.

Reply:
Unlike the table format, this is indeed well-organized and compact, reducing unnecessary white space. It presents all the necessary information clearly and concisely. Good job🥇!